### DISCUSSION TITLE
We are not able to connect to our server since last Steam update

### DISCUSSION THREAD

**Discussion by Lustaa** [mbround18/valheim-docker#195] [Help!] [answered]
Hi, 

Since last update made through steam we are not able to connect to our server anymore.
Docker has been updated as well and auto update set to 1. But still not working.
When connecting, we enter the IP adress. Game launches but instead of prompting for the password, users get disconnected.

Thanks for your help.

**Comment by CosmicHorrorDev** [comment, marked as answer]
Likely your server installation is out of date. Auto updates only check once per day (at the time indicated by your update cron line), so it's likely that this check hasn't happened for you yet

If you have the server volume mounted and look in `server/steamapps/app_manifest_896660.acf` then you should see a line that starts with `"buildid"`. The value next to it _should_ be `"6295625"`, if it is not those value then your server is currently out of date

If your server is out of date then you can create a backup and update with the following

```text
$ docker exec -it --user steam $CONTAINER_NAME bash
steam@hash $ odin stop
steam@hash $ odin backup ~/.config/unity3d/IronGate/Valheim ~/backups/pre-update-manual.tar.gz
steam@hash $ odin install
steam@hash $ odin start
```

This will stop the server, create a backup of your world, install the update, and start the server again.

If you want to be extra cautious then you can manually save a copy of your world from the saves volume mount.

Also I typed all this on mobile so double check the commands instead of just copy-pasting

**Reply by CosmicHorrorDev** [reply]
Oh and if your server is up to date then we can try other troubleshooting, but it's _probably_ out of date

**Reply by Lustaa** [reply]
Thanks, i'll try to update it.

**Reply by CosmicHorrorDev** [reply]
I could have sworn I was just up to date this morning, but it seems like the new current buildid is actually `"6295625"`

**Reply by Lustaa** [reply]
Thanks for helping. Our server works fine now. :)

**Reply by CosmicHorrorDev** [reply]
Woohoo! Enjoy sailing! :D

**Reply by bforbis** [reply]
Hey, I am getting this same "Disconnected" error (ie: not an error that says "incompatible version") and have tried resolving with `odin stop/install/start`, however this has not worked to resolve the issue.

I have found my server `buildid` in the file you mentioned is `6315977`. How do I find if my client buildid is compatible with this server build?

**Reply by bforbis** [reply]
I am seeing a mismatch in current `buildids` in steamdb.info:

Valheim Client: https://steamdb.info/app/892970/depots
- Current build: `6315970`

Valheim Server: https://steamdb.info/app/896660/depots/
- Current Build: `6315977`

Should these be matched up?

**Reply by mbround18** [reply]
not necessarily, i suggest nuking your server installation and letting odin redownload the game.

**Reply by bforbis** [reply]
I think this is the issue I'm running into:
https://www.pcgamesn.com/valheim/dedicated-server-disconnected

**Reply by CosmicHorrorDev** [reply]
Im not sure whether or not those two buildids usually match or not. That would be something good for me to know.

Can you check the currently installed buildid using the process I listed in thr first comment in this thread? It should match the buildid you listed for valheim server if it's up to date

**Reply by bforbis** [reply]
Yep, I have confirmed in my server `server/steamapps/app_manifest_896660.acf` that I have the latest server buildid `6315977`. Same goes with my client, by right clicking the game in steam and going to properties->Updates, and I do have `6315970`.

I have tried restarting the restarting/removing the server container, and have also tried running odin stop / install / update, but nothing has resolved this issue for me yet.

I'm thinking probably just need to wait for Valheim devs to resolve that issue I linked up above, as it seems to be a widespread issue (though not affecting everyone for some reason)
